static/src/javascripts/projects/journalism/modules/audio-series-add-contributions.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+const renderContributionsBanner = el => {
+    const container = document.createElement('section');
+    const fcContainer = document.createElement('div');
+    const fcContainerHead = document.createElement('div');
+    const fcContainerBody = document.createElement('div');
+
+    const header = document.createElement('h2');
+    const text = document.createElement('p');
+    const button = document.createElement('a');
+
+    container.classList.add('flagship-audio__contributions-banner');
```

Comment:
Another fyi, `add` takes a variable amounts of parameters: `add('class1', 'class2', ...)`

Reply:
oh I was told by the linter that pollyfil.io didn't accept this unfortunately